### PR TITLE
Fix Simba specific JDBC tests

### PR DESCRIPTION
**Pull request by brian-rickman**
Adjust tests to accomodate recent changes in data types
and metadata.

### CODE REVIEW

File: presto-product-tests/src/main/resources/com/facebook/presto/tests/jdbc/get_nation_columns_simba.result [hunk 1]
```diff
@@ -1,5 +1,5 @@
 -- delimiter: |; ignoreOrder: false; ignoreExcessRows: false;
 hive|default|nation|n_nationkey|-5|bigint|19|8|0|10|1|null|null|null|null|null|1|YES|null|null|null|null|null|null|
-hive|default|nation|n_name|12|varchar|2048|2048|null|10|1|null|null|null|null|2048|2|YES|null|null|null|null|null|null|
+hive|default|nation|n_name|12|varchar(25)|25|25|null|10|1|null|null|null|null|2048|2|YES|null|null|null|null|null|null|
```
Comment (cawallin) [inline presto-product-tests/src/main/resources/com/facebook/presto/tests/jdbc/get_nation_columns_simba.result:3]: is varchar(25) what is supposed to be printed out for this metadata? or is it varchar with a separate field being 25 (precision)?

File: presto-product-tests/src/main/resources/com/facebook/presto/tests/jdbc/get_nation_columns_simba4.result [hunk 2]
```diff
@@ -1,5 +1,5 @@
 -- delimiter: |; ignoreOrder: false; ignoreExcessRows: false;
 hive|default|nation|n_nationkey|-5|bigint|19|8|0|10|1|null|null|null|null|null|1|YES|null|null|null|null|null|
-hive|default|nation|n_name|12|varchar|2048|2048|null|10|1|null|null|null|null|2048|2|YES|null|null|null|null|null|
+hive|default|nation|n_name|12|varchar(25)|25|25|null|10|1|null|null|null|null|2048|2|YES|null|null|null|null|null|
```
Comment (cawallin) [inline presto-product-tests/src/main/resources/com/facebook/presto/tests/jdbc/get_nation_columns_simba4.result:3]: ditto on varchar(25)

**Review by cawallin — CHANGES_REQUESTED** [review]

File: presto-product-tests/src/main/resources/com/facebook/presto/tests/jdbc/get_nation_columns_simba4.result (hunk 2 above)
Comment (brian-rickman) [reply presto-product-tests/src/main/resources/com/facebook/presto/tests/jdbc/get_nation_columns_simba4.result:3]: That is coming from Presto.  We see the same value with the open source JDBC driver.  I'm not sure why it was changed from `varchar`, but my reading of the docs is that either value is valid:

[JDBC spec](https://docs.oracle.com/javase/7/docs/api/java/sql/DatabaseMetaData.html#getColumns(java.lang.String,%20java.lang.String,%20java.lang.String,%20java.lang.String)) says:
> TYPE_NAME String => Data source dependent type name, for a UDT the type name is fully qualified
> COLUMN_SIZE int => column size.

**Review by cawallin — APPROVED** [review]

**Comment by brian-rickman** [comment]
Adding the changes from #489 to this branch as they are closely related.

**Pull request merged**